### PR TITLE
Hatch holder

### PR DESCRIPTION
**Pull request by Mboesn**

### CODE REVIEW

File: src/main/java/frc/robot/RobotMap.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package frc.robot;
+
+public class RobotMap {
+    public final static int HATCH_HOLDER_FRONT_SOLENOID = 1;
+    public final static int HATCH_HOLDER_RIGHT_BACK_SOLENOID = 2;
+    public final static int HATCH_HOLDER_LEFT_BACK_SOLENOID = 3;
+}
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/RobotMap.java]: names: HATCH_HOLDER_CATCH_SOLENOID, HATCH_HOLDER_RIGHT_PUSH_SOLENOID, HATCH_HOLDER_LEFT_PUSH_SOLENOID

File: src/main/java/frc/robot/SubSystem/HatchHolder.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package frc.robot.SubSystem;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class HatchHolder extends Subsystem {
+  Solenoid frontSolenoid, rightBackSolenoid, leftBackSolenoid;
+  public HatchHolder(Solenoid frontSolenoid, Solenoid rightBackSolenoid, Solenoid leftBackSolenoid){
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/SubSystem/HatchHolder.java]: change class to DoubleSolenoid (not Solenoid) and all the functions in this class acording to this calss...
names: catchSolenoid, rightPushSolenoid, leftPushSolenoid

**Review by HilelAmiel — CHANGES_REQUESTED** [review]
DoubleSolenoid not Solenoid

File: src/main/java/frc/robot/SubSystems/HatchHolder.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    this.rightPushSolenoid = rightPushSolenoid;
+    this.leftPushSolenoid = leftPushSolenoid;
+  }
+
+  /**
+   * sets the status of the front solenoid, the one that chatches the Hatch.
+   * DoubleSolinoids require "values" that represent off, forward, and reversed.
+   */
+  public void setCatch(Value value) {
+    this.pvcSolenoid.set(value);
```
Comment (esh4) [inline src/main/java/frc/robot/SubSystems/HatchHolder.java:23]: DoubleSolenoid.Value

File: src/main/java/frc/robot/SubSystems/HatchHolder.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  /**
+   * sets the status of the rear solenoids, the ones that pushes the SS outwards.
+   * DoubleSolinoids require "values" that represent off, forward, and reversed
+   */
+  public void setPush(Value value) {
+    this.leftPushSolenoid.set(value);
```
Comment (esh4) [inline src/main/java/frc/robot/SubSystems/HatchHolder.java:31]: DoubleSolenoid.Value

**Review by esh4 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/SubSystems/HatchHolder.java (hunk 3 above)
Comment (esh4) [reply src/main/java/frc/robot/SubSystems/HatchHolder.java:23]: cool

**Review by esh4 — APPROVED** [review]

**Pull request merged**